website/blog/2022-08-08-aluminium.md

```diff
@@ -0,0 +1,377 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.8 - Aluminium
+authorURL: https://twitter.com/tanishiking
+authorImageURL: https://github.com/tanishiking.png
+---
+
+We're happy to announce the release of Metals v0.11.8, which brings a number of improvements for both Scala 2 and Scala 3.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">81</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">77</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">15</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">22</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">6</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/52?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Type completion
+- [Scala 3] Auto import and completion for extension methods
+- [Scala 3] Convert to Named Parameters code action
+- [Scala 3] Scaladoc Completion for Scala3
+- [Scala 3] Completions in string interpolation
+- [Scala 2] Automatically import types in string interpolations
```

Comment:
```suggestion
- [Scala 2] Automatic import of types in string interpolations
```
because all the previous items are phrases, rather than sentences

NIT